src/Illuminate/Validation/Rules/AnyOf.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Illuminate\Validation\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Support\Str;
+
+class AnyOf implements Rule
+{
+    public function __construct(protected $rules)
```

Comment:
Could you please include the PHP Docs annotations that we typically have in these types of classes on this pull request or any other methods within it?

Reply:
sure

Reply:
I was still working on that, and that's why the PR is still on draft. But adding all that shortly